src/index.d.ts

```diff
@@ -29,6 +29,8 @@ declare namespace RamdaAdjunct {
 
     interface Dictionary<T> { [key: string]: T; }
 
+    type DictPred<T> = (value: T, key: string) => boolean;
```

Comment:
Just realized, it doesn't take into account the 3rd parameter, probably better would be something like
```typescript
type DictPred<T, U extends Dictionary<T>> = (value: T, key: string, dictionary: U) => boolean;
```

Reply:
You right, and I think `key` and `dictionary` arguments should be optional.
Would you want to PR ?

@char0n : should we make a v2.6.1 patch ?

Reply:
woops, sorry, 2.6.0 is not released yet 😂